### DISCUSSION TITLE
[求助] 请阻止他们找回我的暑假作业

### DISCUSSION THREAD

**Discussion by kishilong** [share121/inter-knot#422] [其他]
![绳网](https://github.com/user-attachments/assets/8ce104c3-90af-445f-ad35-736c7b71848f)

上周我家出现了空洞，逃出来后我的父母想要委托绳匠找回我的暑假作业，请帮我阻止绳匠或者是找到并销毁我的寒假作业，急急急急急

**Comment by IceColaaa** [comment]
这个委托，我接了！

**Comment by kishilong** [comment]
如果一个绳匠同时接到我和我父母的委托该怎么办（思考

**Comment by yiwangzj** [comment]
你的作业上没写名字吧？到时候死不认账好了

**Comment by lbghhj** [comment]
![IMG_20240810_093551](https://github.com/user-attachments/assets/a627ae12-1efc-4f09-b11d-be82d7e82654)

这个委托我笑了！
